packages/core/src/modules/proofs/services/ProofService.ts

```diff
@@ -905,7 +907,7 @@ export class ProofService {
    *
    */
   public async verifyProof(proofJson: IndyProof, proofRequest: ProofRequest): Promise<boolean> {
-    const proof = JsonTransformer.fromJSON(proofJson, PartialProof)
+    const proof = await JsonTransformer.fromJSON(proofJson, PartialProof, { validate: true })
```

Comment:
I see you added `{validate: true}` to every function call, but it is also the default value. I'd suggest to rely on the default value to keep it a bit more terse.